browser/scripts/abstractObjectGazePlugin.js

```diff
@@ -182,8 +182,10 @@
 
 		findSelectableMeshNodes()
 		.map(function(selectable) {
+			var nodeRef = selectable.node.parent_graph.uid + '.' + selectable.node.uid
```

Comment:
the plugin needs to know when stuff changes in the graph, to update the selector